docs/connectors/finance-app.md

```diff
@@ -4,17 +4,36 @@ This is an app connector for the Finance app (`finance.aragonpm.eth`). It only s
 
 ## Usage
 
-To connect an app, you need to pass a finance app as a first parameter of the connectFinance() function:
+To connect a Finance app, you need to pass it to `connectFinance()`:
 
 ```js
 import connect from '@aragon/connect'
 import connectFinance from '@aragon/connect-finance'
 
-const org = connect('myorg.aragonid.eth', 'thegraph')
+const org = await connect('myorg.aragonid.eth', 'thegraph')
 const finance = await connectFinance(org.app('finance'))
 ```
 
-It extends the `App` object, which means that every method and properties of [`App`](../api-reference/app.md) are also available on this object.
+It extends the `App` object, which means that every method and property of [`App`](../api-reference/app.md) is also available on this object.
+
+## connect\(app, connector\)
+
+Connects and returns a `Finance` instance.
+
+| Name        | Type                                   | Description                                                                                                                                            |
+| ----------- | -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `app`       | `App` or `Promise<App>`                | The app to extend with connected capabilities.                                                                                                         |
+| `connector` | `["thegraph", Object]` or `"thegraph"` | Accepts either a string describing the desired connector (only `"thegraph"` for now), or a tuple to also pass a configuration object to the connector. |
+| returns     | `Promise<Finance>`                      | An `Finance` instance (see below).                                                                                                                      |
+
+It can throw the following errors:
+
+| Error type                                                     | Description                                                                                                                      |
+| -------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| [`ErrorInvalidApp`](./errors.md#error-invalid-app)             | Either the passed value is not a valid app object, or its name is not `finance` (`err.reason === "wrong-name"`).                 |
+| [`ErrorInvalidConnector`](./errors.md#error-invalid-connector) | Either the connector configuration format is not valid, or the connector name is not supported (`err.reason === "unsupported"`). |
+| [`ErrorInvalidConnector`](./errors.md#error-invalid-connector) | The connector is not supported.                                                                                                  |
```

Comment:
Is there a difference with the above error?

Reply:
Technically yes, it could be thrown from either [`createAppConnector()`](https://github.com/aragon/connect/blob/errors/packages/connect-core/src/utils/app-connectors.ts#L55-L59) directly, or [from the app connector](https://github.com/aragon/connect/blob/errors/packages/connect-finance/src/connect.ts#L19-L23) implementation.

But your comment made me realize this distinction doesn’t really matter for consumers, I removed it :smile: